newsroom/reports/reports.py

```diff
@@ -140,33 +136,29 @@ def get_product_stories():
 def get_company_report():
     """Returns products by company"""
     results = []
-    company_products = {}
-    companies = get_entity_dict(query_resource("companies"))
-    products = query_resource("products")
+    companies = list(query_resource("companies"))
+    products_data = get_entity_dict(query_resource("products"))
```

Comment:
Same as above, using a map of ProductID -> Product would help here

Reply:
here we don't need because of `get_entity_dict `. this function returns in that format
 `{product["_id"]: product for product in products}`